crates/types/src/field_color.rs

```diff
@@ -21,7 +21,7 @@ impl FieldColor {
         if red_chromaticity > self.red_chromaticity_threshold
             || blue_chromaticity > self.blue_chromaticity_threshold
             || green_chromaticity < self.lower_green_chromaticity_threshold
-            || rgb.g < self.green_luminance_threshold
+            || (rgb.g as f32) < self.green_luminance_threshold
```

Comment:
Because `self.green_luminance_threshold` is now a `f32` and not an `u8` anymore. Therefore I thought we could compare in `f32`. We could also compare in `u8` but does that matter?

Reply:
Ah, I see. I was wondering why you changed the `self.green_luminance_threshold` type but I already found the reason. Resolving.